project-base/app/templates/Admin/Content/OrderStatus/listGrid.html.twig

```diff
@@ -1,39 +0,0 @@
-{% extends '@ShopsysFramework/Admin/Grid/Grid.html.twig' %}
-
-{% block grid_action_cell_type_delete %}
-    {% if row.os.type == TYPE_NEW %}
-        <span title="{{ 'This status can\'t be deleted. This is automatically assigned status to all new orders.'|trans }}"
-            class="js-tooltip cursor-help in-icon in-icon--info" data-toggle="tooltip" data-placement="top"
-        >
-            {{ icon('info') }}
```

Comment:
The parent uses `<i>` tag to render the icon. Shouldn't it be changed in the parent template?

Reply:
I was looking into it and I found out that these notations are equivalent. However, I changed it to `{{ icon('info') }}` in the parent template in [cbee014](https://github.com/shopsys/shopsys/pull/3157/commits/cbee014426bd5aa977833fe8eba45a5b42edf197)